tests/bugfixes/github/test_issue_1481.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+
+@system_tests.CopyFiles("$data_path/exiv2-empty.jpg")
+class CheckXmpLangAltValues(metaclass=system_tests.CaseMeta):
+
+    url = "https://github.com/Exiv2/exiv2/issues/1481"
+
+# Python unittest is filtering out empty pairs of "" and flags up an error if 
```

Comment:
Would it be possible to test the method `LangAltValue::read()` in unit tests instead of the python tests? Maybe there you have more control to use different combinations of quotes.

Reply:
I will look into this today.

Reply:
Your suggestion was helpful. I have added code to the unitTests and removed the Python version.